great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -269,7 +269,7 @@ def get_or_create_spark_session(
         """  # noqa: E501 # FIXME CoP
         spark_config = spark_config or {}
 
-        spark_session: pyspark.SparkSession
+        spark_session: pyspark.SparkSession | pyspark.SparkConnectSession
```

Comment:
Note sure what I did to anger mypy, but we started failing type check because of the assignment on 274 🤷